### ISSUE TITLE
500: Application Error : Admin Dashboard : Solution ?

### ISSUE THREAD

**Issue by flobe1** [dappur/framework#20]
First of all, thank you for this project.
I test and i learn and i have an 
"Error 500 ..... Unable to find template "dashboard.twig" (looked into: ../app/views/dappur). "
Yes, of course this file is in "../app/views/dashboard"...

My solution :
In file app/bootstrap/dependencies.php, line 91, i changed this :
```
if (substr($container['request']->getUri()->getPath(), 0, 10 ) === "dashboard") {
```
into this
```
if (strpos($container['request']->getUri()->getPath(), 'dashboard' ) !== false) {
```

I hope this will help.
Cordially,

**Comment by jeffmthomas** [comment]
Thank you for this fix. I'm in the process of getting ready to a release a major update that includes the blog components and will include this in the fixes.

**Comment by flobe1** [comment]
Hi Jeffmthomas,
I have already done that, and improve it with tree hierarchical view.
I saw and fix a lot of bugs...
I will improve as much as possible.
Cordially.

**Comment by jeffmthomas** [comment]
That's great! I'd love to see your fixes and please report any bugs as you find them.

**Comment by jeffmthomas** [comment]
This was fixed and released with v2.1
